site/docs/components/progress-stepper.mdx

```diff
@@ -8,7 +8,6 @@ needToKnow:
 - Progress steppers may be gated or ungated.
 - A step may show validation feedback.
 - Aim for 3–4 steps.
```

Comment:
Just to clarify - these were removed because the component is deprecated?

Reply:
I actually forgot that this [hasn't been deleted yet](https://github.com/cultureamp/kaizen-design-system/pull/1789). I might restore this.